.github/workflows/coral-pr-pipeline.yml

```diff
@@ -0,0 +1,95 @@
+name: Coral PR Pipeline
+
+on:
```

Comment:
Please add a:

```
permissions
  contents: read
```

here so that this workflow does not have any write permissions.